packages/editor/src/client/panel/publishPagePanel.tsx

```diff
@@ -72,15 +76,15 @@ export function PublishPagePanel({
         {pendingPublishDate && (
           <Box marginBottom={Spacing.Small}>
             <Typography variant="subtitle1" color="alert">
-              There is already a pending publication scheduled at{' '}
+              {t('There is already a pending publication scheduled at')}{' '}
```

Comment:
You should use formatting for this whole Typography block => https://www.i18next.com/translation-function/formatting